packages/compile-solidity/src/profiler/loadParser.ts

```diff
@@ -32,12 +36,15 @@ async function loadParser({ events, compilers: { solc: solcConfig } }) {
   }
 
   // determine normal solc version and then load that version as solcjs
-  const { version } = semver.coerce(solc.version());
+  const result: SemVer | null = semver.coerce(solc.version());
```

Comment:
Is the return type of `semver.coerce` strictly `Semver | null`? If not, you likely either want `Optional<SemVer>` or `SemVer?` here.

Reply:
Yup, it is strictly `Semver | null`. https://github.com/npm/node-semver/blob/main/functions/coerce.js